pkg/virt-controller/watch/application_test.go

```diff
@@ -40,6 +46,38 @@ func newValidGetRequest() *http.Request {
 var _ = Describe("Application", func() {
 	var app VirtControllerApp = VirtControllerApp{}
 
+	Describe("Reinitialization conditions", func() {
+		table.DescribeTable("Re-trigger initialization", func(hasCdiAtInit bool, addCrd bool, removeCrd bool, expectReInit bool) {
```

Comment:
won't fight for this one, but maybe it would improve readability if you introduced helper structure to pass these bool arguments

```
type conditionsTest struct {
	HasCdiAtInit bool
	...
}